misc/scripts/install-local.sh

```diff
@@ -390,32 +383,29 @@ case "$url" in
 	*.zip)
 		download "$url"
 		# hash the file
-		hashcheck "${url##*/}"
-		if [[ $? -ne 0 ]]; then
+		if ! hashcheck "${url##*/}"; then
 			return 1
 		fi
 		# unzip file
 		sudo unzip -q "${url##*/}" 1>&1 2>/dev/null
 		# cd into it
-		cd ./*/ 2> /dev/null
+		cd ./*/ 2> /dev/null || ( fancy_message warn "Couldn't enter into the downloaded archive" && exit 1)
```

Comment:
no smasher thingys. It should be `could not`

Reply:
They are called contractions. Refactoring now.

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/0691fb5e7635a168f4f677bc139b3ec1a9983765

Reply:
> They are called contractions. Refactoring now.

Yeah, I guess...